internal/postgres/hba.go

```diff
@@ -157,3 +152,45 @@ func (hba *HostBasedAuthentication) String() string {
 	return strings.TrimSpace(fmt.Sprintf("%s %s %s %s %s %s",
 		hba.origin, hba.database, hba.user, hba.address, hba.method, hba.options))
 }
+
+// OrderedHBAs is an append-only sequence of pg_hba.conf lines.
+type OrderedHBAs struct {
+	records []string
+}
+
+// Append renders and adds pg_hba.conf lines to o. Nil pointers are ignored.
```

Comment:
Just checking: it wouldn't be worth filtering out the nils before the `grow`, would it?

Reply:
🤔 I might have added the filtering after I had used Grow. I think I'll skip Grow for now.